<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navbar</class>
 <widget class="QWidget" name="Navbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border: 0;
	border-right: 1px solid #ebebeb;
	padding-left: 10px;
	padding-right: 10px;
}

#widget {
	background-color: #f7f7f7;
	background-color: #fafafa;
	border-bottom: 1px solid #ebebeb;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,1,0">
      <item>
       <widget class="QToolButton" name="btnSearch">
        <property name="toolTip">
         <string>Search</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 2px;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/magnifier.png</normaloff>:/img/magnifier.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="toolTip">
         <string>Save</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSaveAll">
        <property name="toolTip">
         <string>Save All</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/storage.png</normaloff>:/img/storage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnReload">
        <property name="toolTip">
         <string>Reload</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/arrow_circle_right.png</normaloff>:/img/arrow_circle_right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRename">
        <property name="toolTip">
         <string>Rename selected files</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/folder_64.png</normaloff>:/img/folder_64.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSettings">
        <property name="toolTip">
         <string>Settings</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/spanner.png</normaloff>:/img/spanner.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSync">
        <property name="toolTip">
         <string>Synchronize to XBMC</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/xbmc.png</normaloff>:/img/xbmc.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnExport">
        <property name="toolTip">
         <string>Export Database</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/export.png</normaloff>:/img/export.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAbout">
        <property name="toolTip">
         <string>About</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
	border: 0;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="FilterWidget" name="filterWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDonate">
        <property name="toolTip">
         <string>Donate</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
	border: 0;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MediaElch.qrc">
          <normaloff>:/img/heart.png</normaloff>:/img/heart.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterWidget</class>
   <extends>QWidget</extends>
   <header>smallWidgets/FilterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
